acceptance/tests/facts/el.rb

```diff
@@ -10,6 +10,8 @@
 # Don't test on RedHat 4, as is does not include yum (See BKR-512)
 confine :to, :platform => /el-[5-7]|centos-[4-7]/
 
+@ip_regex = /^([1-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(\.([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){3}$/
```

Comment:
Why not go with something like https://www.safaribooksonline.com/library/view/regular-expressions-cookbook/9780596802837/ch07s16.html? Or are you specifically disallowing `0.0.0.0` (as an example).